docs/source/sci_api_tutorial/dipole.rst

```diff
@@ -0,0 +1,154 @@
+==============================================
+Bond Dipole
+==============================================
+
+.. note::
+
+    | **This bond_dipole calculation requires you to have 4 CPUs**
+    | **This bond_dipole calculation requires Multiwfn installed**
+
+.. note::
+
+    Science API is a special concept in EnzyHTP. They stands for those top-layer APIs
+    the are supposed to be used in assembling the `workflow <https://enzyhtp-doc.readthedocs.io/en/latest/sci_api_tutorial/how_to_assemble.html#find-the-science-api-that-directly-gives-what-you-need>`_.
+
+**You will find a detailed tutorial of how to use this Science API.**
+
+Input/Output
+=========================================================================================
+
+.. panels::
+
+    :column: col-lg-12 col-md-12 col-sm-12 col-xs-12 p-2 text-left
+
+    .. image:: ../../figures/dipole_bond.svg
+        :width: 100%
+        :alt: dipole_bond
```

Comment:
Please only have input data instead of all configs here. (you can tell by all the required arguments)

Reply:
Please update this .svg